crates/routes/src/feeds.rs

```diff
@@ -505,7 +505,7 @@ fn create_post_items(posts: Vec<PostView>, settings: &Settings) -> LemmyResult<V
     };
 
     let i = Item {
-      title: Some(p.post.name),
+      title: Some(format!("[{0}] {1}", category.name, p.post.name)),
```

Comment:
Just use the `community.name` . Also you don't need the number `{0}`, just use `{}` .

Reply:
Done. Thanks for guiding me along. 🙂